jsprit-examples/src/main/java/com/graphhopper/jsprit/examples/MaximumDistanceConstraintExample.java

```diff
@@ -44,55 +45,56 @@
 
 //import jsprit.core.problem.solution.route.state.StateFactory; //v1.3.1
 
-public class AdditionalDistanceConstraintExample {
+public class MaximumDistanceConstraintExample {
```

Comment:
Please do not override an existing example. If you want to illustrate the feature with an example, you should add a new one.